### PR TITLE
10821 more cloning

### PR DESCRIPTION
**Pull request by smoyth**

### CODE REVIEW

**Comment by smoyth** [comment]
@cooperka I'll fix CI later today

File: app/models/cloning/importer.rb [hunk 1]
```diff
@@ -15,9 +15,20 @@ def import
...
         Zip::InputStream.open(zip_file) do |io|
+          index = 0
           while (entry = io.get_next_entry)
-            klass = entry.name.sub(/.csv$/, "").tr("_", ":").constantize
-            klass.copy_from(io)
+            class_name = entry.name.match(/\A(\w+)-\d+\.csv\z/)[1]
+            klass = class_name.sub(/.csv$/, "").tr("_", ":").constantize
+            tmp_table = "tmp_table_#{index}"
+            SqlRunner.instance.run("CREATE TEMP TABLE #{tmp_table}
+              ON COMMIT DROP AS SELECT * FROM #{klass.table_name} WITH NO DATA")
```
Comment (cooperka) [inline app/models/cloning/importer.rb:24]: Neat 🤔

**Review by cooperka — APPROVED** [review]
LGTM, I'll include this in the deploy 👍

**Comment by cooperka** [comment]
(Waiting on CI to merge)

**Pull request merged**